.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,20 @@
+name:  Publish Package to npm registry
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  publish-npm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          registry-url: https://registry.npmjs.org/
+          scope: '@trans.eu'
+      - run: npm ci
+      - run: npm publish --access public
```

Comment:
> To publish an organization scoped package as public, use npm publish --access public
https://docs.npmjs.com/creating-and-publishing-an-organization-scoped-package#publishing-a-public-organization-scoped-package